src/App.css

```diff
@@ -275,3 +178,119 @@ CSS Organization reference: https://gist.github.com/awkale/ad46e2ade70e833fa178
 .icon__mark {
   fill: currentColor;
 }
+
+.text-field {
+  display: block;
+  width: 100%;
+  height: 3rem; /* after switching to rems, need this or clear button slightly off */
+  padding: 0.25rem 0.75rem;
+  box-sizing: border-box;
+
+  font-family: inherit;
+  font-size: inherit;
+  line-height: 1.75;
+  color: inherit;
+
+  background: var(--input-bg-color);
+  border: var(--border-width) solid var(--input-border-color);
+  border-radius: var(--ui-border-radius);
+  outline: none;
+  box-shadow: none;
+}
+
+.text-field:focus {
+  border-color: var(--charcoal);
+}
+
+.text-field_has-error {
+  margin: 0 0 0.375rem;
+  border: var(--border-width) solid var(--warning-color);
+}
+
+.error {
+  margin: 0.625rem 0;
+  color: var(--warning-color);
+}
+
+.error_type_field {
+  margin: 0 0 1.25rem 0;
+  font-size: 1rem;
+}
+
+@media only screen and (min-width: 600px) {
+  .error_type_field {
+    font-size: 1.125rem;
+  }
+}
+
+@media only screen and (min-width: 1200px) {
+  .error_type_field {
+    font-size: 1.25rem;
+  }
+}
```

Comment:
Do we want to consider making these min-width into CSS variables in case they ever change?

It also might be good to start putting comments in App.css to group the variables together with headings. I appreciate the comments in App.css explaining the purpose of the variables!

Reply:
@connietran-dev Since these are media queries (and pretty standard breakpoints) I hesitate to make them variables because that would encourage us to think of them as easily changeable. Changing the media query breakpoint would require testing every single place it occurs in the browser again! If we have to change them, I'd rather just search replace and do it very mindfully while extensively checking my layout.

Re: putting headings, I totally agree some of the CSS files are getting unwieldy. The only reason I haven't put in header type comments is because I'm planning to ask in office hours about the affect on performance of splitting them into different files with @imports, because I'm not 100% sure how React handles CSS in the end build. But we will definitely come back to that and do something to break it up better.